slyd/app/controllers/spider.js

```diff
@@ -226,8 +226,13 @@ export default BaseController.extend({
     },
 
     updateExtractedItems: function(items) {
-        var extractedItems = items.map(this.wrapItem, this);
-        this.set('extractedItems', extractedItems);
+        this.get('slyd')
+        .loadItems()
+        .then(function(updatedItemDefinitions) {
```

Comment:
This will fetch `items` for every page load request.

It would be better to have the spider controller and the template controller share a base controller (as controllers are singletons) and have them share items through that.